packages/bigtable/src/table.js

```diff
@@ -757,16 +779,39 @@ Table.prototype.mutate = function(entries, callback) {
   entries = flatten(arrify(entries)).map(Mutation.parse);
 
   var grpcOpts = {
-    service: 'BigtableService',
+    service: 'Bigtable',
     method: 'mutateRows'
   };
 
   var reqOpts = {
+    objectMode: true,
     tableName: this.id,
     entries: entries
   };
 
-  this.request(grpcOpts, reqOpts, callback);
+  var stream = pumpify.obj([
+    this.requestStream(grpcOpts, reqOpts),
+    through.obj(function(data, enc, next) {
+      var entries = data.entries.map(function(entry) {
+        return {
+          index: entry.index,
+          status: common.GrpcService.decorateStatus_(entry.status)
+        };
+      });
+
+      next(null, entries);
```

Comment:
Should these be emitted as a group of entries or should each entry be emitted individually?